api/v1alpha1/spinappexecutor_types.go

```diff
@@ -61,7 +61,20 @@ type SpinAppExecutorStatus struct {
 
 // OtelConfig is the supported environment variables for OpenTelemetry
 type OtelConfig struct {
-	Endpoint string `json:"endpoint"`
+	// Sets OTEL_EXPORTER_OTLP_ENDPOINT environment variable
```

Comment:
I still feel like:

```yaml
otel:
  otel_exporter_*
```

feels really uncomfortable to actually write, and i'm not sure what benefit there is to it - @calebschoepp can you elaborate on why you'd prefer this?

Reply:
I like the clarity of spelling out the full env var your setting.

I'd be okay to remove the `otel_` prefix b/c it seems like all OTel env vars have that, but I wouldn't want to remove anything else for the sake of clarity.

Reply:
dropping the `otel_` prefix at least deals with the stutter so that works